src/components/PrimaryButton/PrimaryButton.styles.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+export const PrimaryButtonWrapper = styled.button`
+  &,
+  &:active,
+  &:focus {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: var(--small-space) var(--large-space);
+    border-radius: 5px;
+    background: var(--lightblue);
+    color: var(--lightgrey);
+    font-weight: 700;
+  }
+
+  &:hover {
+    background: var(--logopink);
+    color: var(--lightgrey);
+  }
+`;
```

Comment:
So a few things need to change here:

• Don't use pixel values. We're using `rem` for sizing in this design system.
• It's not in the design but it's bad practice to have your active and focus states look the same as your default styles. These states show whether a user is on the button if they're navigating with a keyboard.
• You don't need flex here. The text will be centere because of the padding you've defined.

So with all the above in mind you should end up with something like this:

```
import styled from 'styled-components';

export const PrimaryButtonWrapper = styled.button`
  padding: var(--small-space) var(--large-space);
  border-radius: 0.5rem;
  background: var(--lightblue);
  color: var(--lightgrey);
  font-weight: 700;

  &:active,
  &:focus {
    outline: 0.1rem dashed var(--lightblue);
    outline-offset: var(--smallest-space);
    -moz-outline-radius: var(--smallest-space);
    background: var(--lightblue);
    color: var(--lightgrey);
  }

  &:hover {
    background: var(--logopink);
    color: var(--lightgrey);
  }
`;

Let me know if any of that doesn't make sense, happy to go through in more detail :) 
```